aeron-driver/src/main/c/aeron_driver_context.c

```diff
@@ -973,6 +977,20 @@ int aeron_driver_context_init(aeron_driver_context_t **context)
         1,
         AERON_NETWORK_PUBLICATION_MAX_MESSAGES_PER_SEND);
 
+    _context->resource_free_limit = aeron_config_parse_uint32(
+        AERON_DRIVER_RESOURCE_FREE_LIMIT_ENV_VAR,
+        getenv(AERON_DRIVER_RESOURCE_FREE_LIMIT_ENV_VAR),
+        (uint32_t)_context->resource_free_limit,
+        1,
+        UINT32_MAX);
+
+    _context->resource_free_limit = aeron_config_parse_uint32(
```

Comment:
Config option `resource_free_limit` is assigned twice. Should it be `resource_free_queue_length` instead?

Reply:
Yes it should.